content/en/code_analysis/faq/_index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Code Analysis Frequently Asked Questions
+description: What are the frequently asked questions about Datadog Code Analysis
+further_reading:
+- link: "/code_analysis/"
+  tag: "Documentation"
+  text: "Learn about Code Analysis"
+- link: "/code_analysis/static_analysis/"
+  tag: "Documentation"
+  text: "Learn about Static Analysis"
+- link: "/code_analysis/software_composition_analysis/"
+  tag: "Documentation"
+  text: "Learn about Software Composition Analysis"
+---
+
+## Common questions
+
+### What are Datadog-hosted scans?
+
+When using a Datadog-hosted scan, Datadog scans your code within the Datadog infrastructure. Datadog acts
+as a CI pipeline: it clones your code, runs the analyzer, and uploads the results for you.
+
+With Datadog-hosted scans, you do not need to do the heavy lifting and configure any CI pipeline to run the analyzer.
+
+## Static analysis
+
+### Can results be imported into Datadog from other analyzers?
+
+While the [Datadog Static Analyzer tool][4] is recommended, Datadog supports the ingestion of files that enforce the SARIF format.
+
+At this time, we tested our API with the following tools:
+
+ - [gitleaks][1]
+ - [semgrep][2]
+ - [eslint][3]
+
+To ingest your SARIF file into Datadog:
+1. Install the `datadog-ci` CLI (requires that Node.js is installed).
+2. Ensure that your `DD_SITE`, `DD_API_KEY`, and `DD_APP_KEY` environment variables are set.
+3. Invoke the tool to upload the file to Datadog.
+
+Installing and invoking the tool can be done with the two following commands:
+
+```bash
+# Install datadog-ci
+npm install -g @datadog/datadog-ci
+
+# Upload SARIF file
+datadog-ci sarif upload --service "your-app" --env "ci" /path/to/sarif-file.json
+```
+
+If you want to import using a tool that is not supported currently, contact your Customer Success Manager.
+
+### Can custom rules be used?
+
+Custom rule availability is planned for all beta customers at the end of Q3 2024.
+
+### Do you always scan all the files at every push or commit?
+
+Scans on the default branches are always full scans.
+
+Scans on non-default branches use *diff-aware* scans. With diff-aware scans, the analyzer only
+scans the files that changed between the current branch and the default branch. Diff-aware scans
+last seconds while full-scans may take a few minutes (depending on the codebase).
+
+In order for diff-aware to scan, you need to be using the Datadog platform.
+
+### What is Datadog Static Analyzer on the OWASP benchmark?
+
+Our Static Analyzer has been tested against the OWASP benchmark with a score of 44. We periodically update the results against the benchmark and publish the results in the [static analyzer documentation][5].
```

Comment:
Datadog's Static Analyzer has been tested against the OWASP benchmark with a score of 44. The analyzer is periodically checked against the benchmark and updated results are published to the [static analyzer documentation][5].